src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,39 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] NUMBERS = new int[12];
+	private static final int STACK_EMPTY = -1;
+	private static final int STACK_FULL = 11;

```

Comment:
This constant is related to STACK_CAPACITY. Changing only one leads to an error (inconsistency).